independent-publisher-connectors/Doppler/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Doppler Farhan Latif",
+    "description": "Doppler\u2019s developer-first security platform empowers teams to seamlessly manage, orchestrate, and govern secrets at scale.",
+    "version": "1.0",
+    "contact": {
```

Comment:
can you please specify url as mentioned in the template in the below link?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json